exercises/concept/making-the-grade/.docs/introduction.md

```diff
@@ -0,0 +1,96 @@
+# Introduction
+
+## Loops
+
+Loops are used to repeatedly execute some logic.
+The two most common types are the `while loop` (indefinite looping) and the `for loop` (definite, or counted looping).
+There is also the `for each` loop, that will come up in a later concept.
+
+
+### General Syntax
+
+The `for loop` consists of a header and a code block that contains the body of the loop wrapped in curly brackets.
+The header consists of 3 components separated by semicolons `;`:  init-statement, condition, and another expression.
+Each of these may be empty.
+
+```cpp
+for (init-statement; condition; expression) {
+  some_statement;
+}
+```
+
+- The **init** component is some code that runs only once before the loop starts.
+- The **condition** component must be some expression that evaluates to a boolean and controls when the loop should stop.
+The code inside the loop will run as long as this condition evaluates to true.
+As soon as this expression evaluates to false, no more iterations of the loop will run.
+- The **expression** component is some code that will run at the end of each iteration.
+
+
+The `while loop` executes its body as long as its condition check is true.
+The code snippet below shows how to transform a for into a while loop.
+
+```cpp
+init-statement;
+while(condition) {
+    some_statement;
+    expression;
+}
+```
+
+### Interlude: Increments and Decrements
+
+When working with loops is often required to add 1 or subtract 1 from a counter variable.
+This is so common, that the operation has special operators: `++` and `--`.
+
+They come in a prefix and a postfix form.
+The prefix changes the variable before use in the statement and the postfix version afterward.
+You probably want the prefix version most of the time.
+
+```cpp
+int a{3};
+int b{--a};
+// b is 2, a is now 2
+int c{a++};
+// c is 2, a is now 3
+```
+
+### For Loops - An example
+
+The init component usually sets up a counter variable, the condition checks whether the loop should be continued or stopped and the post component usually increments the counter at the end of each repetition.
```

Comment:
Not sure how to emphasize things here, but it _feels_ like init, condition, and post should be highlighted, bulleted or otherwise called out somehow.